app/templates/users/details.html

```diff
@@ -12,8 +18,50 @@
 
 {% block main_column %}
 
+{% if current_user.is_superuser %}
+<form action="{{ url_for('users.edit', { id: user.auth0_id | urlencode }) }}" method="post">
```

Comment:
As I just mentioned in other comment, this probably will need to submit to a new route called `PUT users.update` instead of changing existing route to view edit form.

Reply:
Usually only `GET` and `POST` methods are used by HTML forms, as other verbs may not be supported by browsers (or at least historically). If you look at the other routes in the frontend, there are no "edit pages" for other objects (eg apps) - the forms are displayed on the details page and submit to POST-only update handlers.

Reply:
Yeah I know, that's annoying. I didn't think about that, for some reason I thought we were using some king of fancy wrapper to generate the form code with an hidden support field (like `_method`) or something like that, sorry I'm tired, you're right 👍